lib/hawkular/operations/operations_api.rb

```diff
@@ -30,16 +32,14 @@ module Hawkular::Operations
   class OperationsClient < Hawkular::BaseClient
     include WebSocket::Client
 
-    attr_accessor :ws, :session_id
+    attr_accessor :ws, :session_id, :logger
 
     # helper for parsing the "OperationName=json_payload" messages
     class WebSocket::Frame::Data
       def to_msg_hash
         chunks = split('=', 2)
-        {
-          operationName: chunks[0],
-          data: JSON.parse(chunks[1])
-        }
+
+        { :operationName => chunks[0], :data => JSON.parse(chunks[1]) }
```

Comment:
Could you please tell why changing to this syntax? Also our rubocop is configured to use the Ruby: 1.9 syntax over hash rocket.